pallets/runtime/common/src/bridge.rs

```diff
@@ -326,6 +327,10 @@ decl_event! {
         WhiteListUpdated(IdentityId, IdentityId, bool),
         /// Bridge limit has been updated
         BridgeLimitUpdated(IdentityId, Balance, BlockNumber),
+        /// An event emitted after a vector of transactions is handled. The parameter is a vector of
+        /// failed transactions, each with its failure reason. If there are no failed transactions
+        /// then the vector has been successfully handled.
+        TxsHandled(Vec<(BridgeTx<AccountId, Balance>, Vec<u8>)>),
```

Comment:
Can we rename this to something more clear like `TxsFailedToHandle`

Reply:
Adding `Failed` in the event name might be wrongly interpreted as failure of the batch transaction. Whereas here we emit this even in any case, whether there were failures or not. Moreover, by the event naming convention, an event name is understood as a sentence, so `TxsFailedToHandle` does literally translate as "the transactions failed to handle" which is not what we want.

I'm open to other proposals though.

Reply:
>so `TxsFailedToHandle` does literally translate as "the transactions failed to handle" which is not what we want.

But isn't that exactly what happened? The system failed to handle these transactions for some reason. The event specifically contains an array of tx that failed. That being said, as long as the code comment is clear (which it is), I don't mind.

Reply:
I mean that in case the vector of failed transactions is empty, the event name `TxsFailedToHandle` would still wrongly suggest that the transactions have failed. That's why I think a constructive name (without negations) is better.

Reply:
Why don't we modify to just emit all transactions in the batch, with an error code of 0 for those with no failures.